### PR TITLE
feat: view payment account

### PR DESCRIPTION
**Pull request by jatnerubia**
Ticket ID: [Create - View Payment Account Screen](https://github.com/orgs/nerubia/projects/24/views/2?pane=issue&itemId=84432908&issue=nerubia%7Ckh-books%7C343)

### CODE REVIEW

File: src/services/khbooks/payment-account-service.ts [hunk 1]
```diff
@@ -39,3 +40,17 @@ export const getAllByFilters = async (page: string) => {
...
+
+export const getById = async (id: number) => {
+  return await PaymentAccountRepository.getById(id)
+}
+
+export const deleteById = async (id: number) => {
+  const paymentAccount = await PaymentAccountRepository.getById(id)
+
+  if (paymentAccount === null) {
+    throw new CustomError("Offering not found", 400)
```
Comment (emmaacha) [inline src/services/khbooks/payment-account-service.ts:52]: I think this should be `"Payment Account not found"`

**Review by emmaacha — CHANGES_REQUESTED** [review]
Just a really minor change needed

**Comment by jatnerubia** [comment]
@emmaacha done ^^

**Review by emmaacha — APPROVED** [review]
Cool 🙌

**Pull request merged**